src/main/groovy/de/dkfz/roddy/tools/versions/VersionIO.groovy

```diff
@@ -0,0 +1,43 @@
+package de.dkfz.roddy.tools.versions
+
+class VersionIO {
+
+    static Version fromBuildVersion(List<String> buildVersion) {
+        if (buildVersion.size() == 2) {
+            return new Version(
+                    buildVersion[0].split("\\.")[0].toInteger(),
+                    buildVersion[0].split("\\.")[1].toInteger(),
+                    buildVersion[1].toInteger(),
+            )
+        } else {
+             throw new RuntimeException("Could not parse build information from '${buildVersion}'")
```

Comment:
NumberFormat? Or maybe also a custom class. You throw several exceptions where a custom exception class might be appropriate.